src/ms.tex

```diff
@@ -203,14 +203,14 @@ \subsection{Highlighted Feature Development} \label{sec:core-features}
 astronomy \python community to rely upon the \astropypkg core package and build
 specialized packages within the Astropy ecosystem. Even many non-astronomical
 \python libraries have come to rely on \astropypkg. The relationship between
-\astropy and the \python-using community is reciprocal, with the evolving needs
+\astropy and the broader \python community is reciprocal, with the evolving needs
 of the community and the maturation of the broader scientific \python ecosystem
 driving much of the development of the \astropypkg core package.
 
-Within \astropypkg, development may be roughly split into a few
-categories: new features added, intra- and inter-package interoperability,
-improvements to precision, accuracy, and reproducibility, and performance
```

Comment:
Is there a difference between "precision" and "accuracy"? Do we need both?

Reply:
I think below, "precision" is used to refer to numerical precision, whereas "accuracy" refers to the capacity of algorithms to produce correct values. So I think they are subtly different, but maybe we should change precision -> numerical precision?

Reply:
OK, added it back.